bin/get-hetzner-cloud-token

```diff
@@ -0,0 +1,18 @@
+#!/bin/bash
```

Comment:
In general, I'd prefer this written in Python. And also it looks like there's already a `contrib/` folder for this so no reason to create another one.